### DISCUSSION TITLE
open-meteo exporter for influxdb

### DISCUSSION THREAD

**Discussion by rare-magma** [open-meteo/open-meteo#545] [Show and tell]
Hi,
Thanks for creating this awesome project!
I've put together a bash script that uploads the current air quality and weather data from the Open-Meteo API to influxdb on an hourly basis here: https://github.com/rare-magma/open-meteo-exporter

I use it to populate a grafana dashboard and avoid unnecessary calls to the open-meteo API.

Sharing it here since it might be useful for other people.
Cheers!

**Comment by patrick-zippenfenig** [comment]
Thanks for sharing!
